### PR TITLE
TUI Help/About message customisation point for variants

### PR DESCRIPTION
**Pull request by CarlosNihelton**
WSL installer TUI currently crashes if user clicks on Help menu due missing network parameter in the app basemodel. When fixing that we noticed that the Help/About message required some customisation. This PR aims to provide the customisation point for variants and provide the fix for the WSL case.
This aims to close https://github.com/ubuntu/WSL/issues/202

### CODE REVIEW

File: subiquity/ui/views/help.py [hunk 1]
```diff
@@ -394,8 +394,9 @@ def _close(self, sender):
 
 class HelpMenu(PopUpLauncher):
 
-    def __init__(self, app):
+    def __init__(self, app, about_msg=None):
```
Comment (dbungert) [inline subiquity/ui/views/help.py:397]: This arg isn't used, correct?

Comment (CarlosNihelton) [reply subiquity/ui/views/help.py:397]: Indeed. It could be, though, but I avoided changing the SubiquityClient init code.

Comment (mwhudson) [reply subiquity/ui/views/help.py:397]: Please feel free to modify the subiquity code if it makes sense to do so!

File: subiquity/client/client.py [hunk 2]
```diff
@@ -120,13 +120,13 @@ def make_ui(self):
 
     variant_to_controllers: Dict[str, List[str]] = {}
 
-    def __init__(self, opts):
+    def __init__(self, opts, about_msg=None):
```
Comment (CarlosNihelton) [inline subiquity/client/client.py:123]: This was what I originally wanted, but gave up in the middle of the process and you guys encouraged me doing 😄 
Thanks for that.

**Review by dbungert — APPROVED** [review]
LGTM

**Review by mwhudson — APPROVED** [review]
This looks fine, feel free to merge.

It does make me a bit sad again that system_setup inherits so much from the subiquity package -- really we should move the generic client / server bits out of subiquity to somewhere else and leave subiquity to have the installer specific parts. But well, that's a lot of refactoring. Maybe one day.

**Pull request merged**